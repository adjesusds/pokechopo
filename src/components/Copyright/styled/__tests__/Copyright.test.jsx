// External Packages
import React from 'react'
import { create } from 'react-test-renderer'

// Internal Packages
import StyledCopyright from '../Copyright'

describe('<StyledCopyright />', () => {
  it('should style the component on valid props', () => {
    const styledComponent = create(<StyledCopyright />)

    expect(styledComponent).toMatchSnapshot()
  })
})
