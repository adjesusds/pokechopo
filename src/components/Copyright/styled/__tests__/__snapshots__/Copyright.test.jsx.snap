// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledCopyright /> should style the component on valid props 1`] = `
.c0 {
  bottom: 0;
  font-size: 0.75rem;
  margin: 0.5rem;
  position: absolute;
  right: 0;
}

<span
  className="c0"
/>
`;
